utils/PackageGenerator.ts

```diff
@@ -12,17 +12,21 @@ strOptions.fold.lineWidth = 0;
 
 interface PackageGeneratorOptions extends GeneratorOptions {
     packageName: string;
+    packagePath: string;
 }
 
 class PackageGenerator<T extends PackageGeneratorOptions = PackageGeneratorOptions> extends Generator<T> {
     constructor(args: string | string[], opts: T) {
         super(args, opts);
 
         this.argument('packageName', { type: String, required: true });
+        this.option('path', { type: String });
```

Comment:
should we have any check / validation on the path or are we accepting anything ?